ncm-iptables/src/main/perl/iptables.pm

```diff
@@ -336,54 +344,54 @@ sub upercase {
 #                     - 4 $config is not an object,
 #                     - 5 $path doesn't exist as a resource path,
 #                     - 6 $path has no entries.
-##########################################################################
 sub GetPathEntries {
     my ($path, $config) = @_;
     my ($content, $entry, $name, $value);
     my $entries = {};
+    local $@;
```

Comment:
you need to localize it where method is called, doing this probably breaks the functionality